packages/paste-core/components/button/src/ResetButton.tsx

```diff
@@ -30,6 +30,7 @@ const ResetButton = React.forwardRef<HTMLButtonElement, DirectButtonProps>(
       <Box
         ref={ref}
         width={fullWidth ? '100%' : 'auto'}
+        textAlign="left"
```

Comment:
I'm pretty sure the reset button doesn't have any alignment set, which should default to `inherit` or `initial` iirc. Adding this could cause issues in existing implementations, maybe we should leave it off?